lib/uefi/variable_mem.cpp

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (C) 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include "variable_mem.h"
+
+#include <stdio.h>
+#include <string.h>
+#include <malloc.h>
+#include <lk/list.h>
+#include <uefi/types.h>
+#include "charset.h"
+
+namespace {
+  struct EfiVariableInMem {
```

Comment:
Remove the `InMem` suffix, as structs are inherently in memory, and we don't have persistent variables anyway.